extensions/PhabBugz/lib/Feed.pm

```diff
@@ -237,7 +237,7 @@ sub process_revision_change {
 
     my ($timestamp) = Bugzilla->dbh->selectrow_array("SELECT NOW()");
 
-    my $attachment = create_revision_attachment($bug, $revision->id, $revision->title, $timestamp);
+    my $attachment = create_revision_attachment($bug, $revision, $timestamp);
```

Comment:
suggestion: maybe this should be passed a hash, so you can use named parameters?